package.json

```diff
@@ -87,6 +87,7 @@
         "@polkadot/api-derive": "10.12.4",
         "@polkadot/typegen": "10.12.4",
         "@polkadot/types": "10.12.4",
-        "@polkadot/types-codec": "10.12.4"
+        "@polkadot/types-codec": "10.12.4",
+        "@logion/node-api": "^0.31.0-2"
```

Comment:
Why do you need `@logion/node-api` in the resolutions?

Reply:
To insure only the latest version is used - `yarn dedupe` did not work.